tools/fq.wdl

```diff
@@ -158,13 +158,13 @@ task subsample {
         fq subsample \
             ~{probability_arg} \
             ~{record_count_arg} \
-            --r1-dst ~{r1_dst} \
+            --r1-dst "~{r1_dst}" \
             ~{(
                 if defined(read_two_fastq)
                 then "--r2-dst " + r2_dst
                 else ""
             )} \
-            ~{read_one_fastq} \
+            "~{read_one_fastq}" \
             ~{read_two_fastq}
```

Comment:
This is a shellcheck warning, but the `File` is optional. So I don't think we want to quote it and end up passing `""` to the command.

Reply:
We could do that, but it's a bit ugly compared to the bare statement. It also doesn't resolve the shellcheck warning.
```
note[ShellCheck]: Double quote to prevent globbing and word splitting.
    ┌─ /Users/athrashe/Library/CloudStorage/OneDrive-St.JudeChildrensResearchHospital/repo/workflows/tools/fq.wdl:168:13
    │
168 │             ~{if defined(read_two_fastq) then "\"~{read_two_fastq}\"" else ""}
    │             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    │             │
    │             SC2086[info]: Double quote to prevent globbing and word splitting.
    │             more info: https://www.shellcheck.net/wiki/SC2086
    │
    = fix: address the diagnostic as recommended in the message
```